lib/docker-compose-definitions-quorum.yml

```diff
@@ -69,7 +70,8 @@ x-quorum-def:
         --port ${QUORUM_P2P_PORT:-21000} \
         --unlock 0 \
         --password $${DDIR}/keystore/password.txt \
-        ${QUORUM_GETH_ARGS:-} $${GETH_ARGS_${QUORUM_CONSENSUS:-istanbul}}
+        $${GETH_ARGS_${QUORUM_CONSENSUS:-istanbul}} \
+        $$EXTRA_ARGS
```

Comment:
it was wrapping all of the flags in QUORUM_GETH_ARGS in single quotes if i didn't reassign this to a local env variable before using it